frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTabStatus.tsx

```diff
@@ -25,10 +32,22 @@ const LogsTabStatus: React.FC<LogsTabStatusProps> = ({
       <Alert
         isInline
         variant="danger"
-        title="An error occurred while retrieving the requested logs"
-        actionLinks={<AlertActionLink onClick={refresh}>Retry</AlertActionLink>}
+        title={
+          (error?.message.includes('Failed to fetch') && error?.message + ' logs') ||
```

Comment:
I would abstract that `error?.message.includes('Failed to fetch') ...` logic into some variable or function above in this component because it is being used multiple times in this component. The inline logic is confusing to read otherwise

Reply:
That is better, but i still get lost in the rest of the logic. This is fine for now so i can resolve this comment, but for better readability i would suggest something like this. This component is just this logs and error so you can afford to have a little logic in the main body of the component
```tsx
    const isNetworkError = error?.message.includes('Failed to fetch');
    const podCondition = podStatus === null || (podError && podStatus?.completed);

    let alertTitle;
    let alertMessage;

    if (isNetworkError) {
        alertTitle = `${error?.message} logs`;
        alertMessage = 'Check your connection and reload the page.';
    } else if (podCondition) {
        alertTitle = podError?.message;
        alertMessage = `${podName} may have been pruned to prevent over-utilization of resources.`;
    }
```

Reply:
Oh yes, thanks for pointing this out.